src/AstRunner/AstMap/ClassLikeName.php

```diff
@@ -4,9 +4,6 @@
 
 namespace Qossmic\Deptrac\AstRunner\AstMap;
 
-/**
- * @psalm-immutable
```

Comment:
Why remove this? As far as I can tell this class is still immutable

Reply:
This one, yes. It was on the interface, so the implementors had to have it also, but one of those is not really immutable (FunctionName), so I had to remove it there, so I had to remove it on the interface, so having it here is not very helpful anymore.